app/forms/schools/update_participant_name_form.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Schools
+  class UpdateParticipantNameForm
+    include ActiveModel::Model
+
+    attr_accessor :full_name
+    validates :full_name, presence: true
+  end
```

Comment:
This feels like it's barely doing anything.

Do we want to move towards a pattern of having a form for every data input in our app?